### PR TITLE
feat: added container tokens

### PR DESCRIPTION
**Pull request by GutierrezGo**
References:
> Jira https://autoricardo.atlassian.net/browse/DM-242

> Figma https://www.figma.com/file/WmNmJ7jdB0Z7tOjj516nFB/AS24-Tokens?node-id=84%3A36

## Motivation and context

Setup of base theme CONTAINER tokens as defined in the design system.

## Before

No base theme for the container tokens

## After

Base theme for container tokens 📦

### CODE REVIEW

**Review by talamcol — COMMENTED** [review]
For what are those needed?

File: src/theme/stories/theme.stories.mdx [hunk 1]
```diff
@@ -142,3 +143,26 @@ import BorderRadiusShowcase from './borderRadiusShowcase.tsx';
...
+## Containers
+
+<Canvas>
+  <Story name="containers" args={{ theme: 'as24' }}>
+    <TableContainer>
+      <Table>
+        <Thead>
+          <Tr>
+            <Th>Name</Th>
+            <Th>Rem</Th>
```
Comment (Averethel) [inline src/theme/stories/theme.stories.mdx:156]: This should be pixels, right?

**Review by Averethel — APPROVED** [review]

**Comment by Averethel** [comment]
@talamcol If I understand correctly those are the containers we'd use to set max-width of the content

**Comment by GutierrezGo** [comment]
> For what are those needed?

The containers tokens are the ones we will use to set elements in a max-width of a content.

**Pull request merged**